### PR TITLE
Assessment completion work (ADAPT-957)

### PR DESCRIPTION
**Pull request by barry-learningpool**
Approved by @ryangadams and @bell87 and merged in fork [here](https://github.com/ryan-lynch/adapt-tincan/pull/1)

### CODE REVIEW

**Comment by dennis-learningpool** [comment]
Need to remove all instances of `console.log`

**Comment by brian-learningpool** [comment]
+1

File: js/adapt-xapi.js [hunk 1]
```diff
@@ -331,22 +352,44 @@ define(function(require) {
     },
 
     validateParams: function() {
-      var isValid = true;
-      if (!this.get('actor') || typeof this.get('actor') != 'object') {
-        console.log('\'actor\' is invalid');
-        isValid = false;
+      return !(
```
Comment (dennis-learningpool) [inline js/adapt-xapi.js]: can we get rid of this negation avalanche? It makes my head spin just looking at it. Break this down into a few statements, please.

**Comment by ryangadams** [comment]
+1 from me.

**Comment by dennis-learningpool** [comment]
+1

**Pull request merged**